src/AutoRest.CSharp/Common/Input/InputTypes/InputOperation.cs

```diff
@@ -124,7 +124,7 @@ public static InputOperation RemoveApiVersionParam(InputOperation operation)
     private IReadOnlyList<InputOperationExample> EnsureExamples()
     {
         // see if we need to generate the mock examples
-        if (Configuration.ExamplesDirectory != null)
+        if (Configuration.ExamplesDirectory != null || Configuration.AzureArm)
```

Comment:
The mock example generation is only for DPG?

Reply:
yeah we did not have examples from mock values in MPG because all the mgmt swaggers are required to have real examples.